source/simulator/stokes_matrix_free.cc

```diff
@@ -1391,20 +1392,22 @@ namespace aspect
     // Project the active level viscosity vector to multilevel vector representations
     // using MG transfer objects. This transfer is based on the same linear operator used to
     // transfer data inside a v-cycle.
-    MGTransferMatrixFree<dim,double> transfer;
+    MGTransferMatrixFree<dim,GMGNumberType> transfer;
+
     transfer.build(dof_handler_projection);
-    transfer.interpolate_to_mg(dof_handler_projection,
-                               level_viscosity_vector,
-                               active_viscosity_vector);
+    transfer.template interpolate_to_mg<double>(dof_handler_projection,
```

Comment:
Is this correct then, or shouldn't this use the number type?

Reply:
Yes, it is. I left a comment.